rest-server/src/components/mapbox/mapboxControllers.js

```diff
@@ -0,0 +1,19 @@
+import axios from 'axios';
+import { fetchSingleUserHelper } from '../users/userHelpers';
+
+const mapboxAPIKey = process.env.mapboxAPIKey || 'MUST-REPLACE-WITH-APIKEY-TO-PASS-TEST';
```

Comment:
You can even add the mapboxAPIKey value into your Dockerfile.

Reply:
Cool! I've updated the Dockerfile to reflect that change. Would this still apply if it was an API key that could potentially be billed for? I ask this because I've read about not pushing up your keys to public repositories.